services/workflows-service/src/statistics/statistics-output.dto.ts

```diff
@@ -0,0 +1,45 @@
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+
+export class RiskIndicatorsStatisticsDto {
+  @Type(() => String)
+  name!: string;
+
+  @Type(() => Number)
+  count!: number;
+}
+
+export class ReportsByRiskLevel {
+  @Type(() => Number)
+  low!: number;
+
+  @Type(() => Number)
+  medium!: number;
+
+  @Type(() => Number)
+  high!: number;
+
+  @Type(() => Number)
+  critical!: number;
+}
+
+export class ReportsStatisticsDto {
+  @Type(() => ReportsByRiskLevel)
+  all!: ReportsByRiskLevel;
+
+  @Type(() => ReportsByRiskLevel)
+  inProgress!: ReportsByRiskLevel;
+
+  @Type(() => ReportsByRiskLevel)
+  approved!: ReportsByRiskLevel;
+}
+
+export class StatisticsOutputDto {
+  @ValidateNested({ each: true })
+  @Type(() => RiskIndicatorsStatisticsDto)
+  riskIndicators!: RiskIndicatorsStatisticsDto[];
+
+  @ValidateNested()
```

Comment:
How does `ValidateNested` work with only `Type` decorators? Doesn't it only work with `@IsNumber` and so on?